src/main/java/com/rakovets/course/java/core/practice/exception_handling/exception/DerivativeFromRunTimeException.java

```diff
@@ -0,0 +1,7 @@
+package com.rakovets.course.java.core.practice.exception_handling.exception;
+
+public class DerivativeFromRunTimeException extends RuntimeException{
```

Comment:
"Violation: There isn't separating white spaces (OR line break) for curly brake
Solution: Add white spaces"